### DISCUSSION TITLE
Cant connect on MongoDB Atlas - Receiving Error: querySrv ETIMEOUT _mongodb._tcp.cluster0.m6ivnwl.mongodb.net

### DISCUSSION THREAD

**Discussion by mateussoutolima** [FlowiseAI/Flowise#1596] [Q&A]
I'm trying to connect my flow with MongoDB using a sample dataset just to test it if it is all good and I keep getting this error message 
![xxx](https://github.com/FlowiseAI/Flowise/assets/87151987/7f99d3bf-688c-485e-a2db-9f8c9c721ae6)

Can anyone help me fix it? I REALLY new at this

Thanks in advance!

**Comment by Jaredude** [comment]
Is the MongoDB database setup to accept connections from all outside networks? Or at least the server you're making the request from?

**Reply by mateussoutolima** [reply]
Well, I think it is. I'm on a local enviroment and I put my IP address on mongo as you can see in the printscreen below
![image](https://github.com/FlowiseAI/Flowise/assets/87151987/4a9d0c59-f54a-4942-a0d3-f8b8a9bcb5d1)

I'm not sure about what to put in the credential name in the node: 
![image](https://github.com/FlowiseAI/Flowise/assets/87151987/5bb7c31b-770f-4e48-837f-d5ee1cada163)
This is the one that I'm using it

And for my connection test I'm using the airbnb sample data avaiable in Mongo

![image](https://github.com/FlowiseAI/Flowise/assets/87151987/2972a466-e1b3-405f-8f95-9784a6bd3343)

![image](https://github.com/FlowiseAI/Flowise/assets/87151987/2dfa727f-6d36-43ec-9a07-0e7d48b9c819)

![image](https://github.com/FlowiseAI/Flowise/assets/87151987/6db99456-6f4b-4ba8-bd6a-0db6ab71b3a1)

Still getting the same message

**Reply by HenryHengZJ** [reply]
For mongo, there is some setup required https://js.langchain.com/docs/integrations/vectorstores/mongodb_atlas
